rfcs/20191212-keras-categorical-inputs.md

```diff
@@ -1,15 +1,15 @@
 # Keras categorical inputs
 
-| Status        | Accepted                                             |
+| Status        | Completed                                            |
```

Comment:
Do you mean it's implemented?  I would prefer the terminology Implemented instead of Completed.  Could you also post the PR number where it was implemented?  Something like:

Status: Implemented (Link to PR)

Reply:
Done.